src/lib/db.js

```diff
@@ -315,6 +320,40 @@ export default class db {
       })
     })
   }
+  
+  teamArrayQuery() {
```

Comment:
No corresponding falcor route for this at the moment, and this is one that Zane might need to use